### ISSUE TITLE
Zero shot classification 

### ISSUE THREAD

**Issue by rom1504** [kakaobrain/coyo-dataset#13]
Hello, did you train clip models on coyo and  evaluate them on zero shot classification ?

I'm curious to know what zero shot classification on imagenet numbers you got

**Comment by justHungryMan** [comment]
@rom1504 We did not evaluate clip trained with coyo on zero shot classification.

I think the person who asking the question got confused with knn results. (imagenet)

**Comment by yaoyu-33** [comment]
Yes you are right. We are still trying to figure out if the results difference comes from the dataset. Looks like it has the highest possibility. I was thinking imagenet knn and zeroshot might be somewhat transferrable but might be just too arbitrary.
